status/status.go

```diff
@@ -0,0 +1,29 @@
+package status
+
+// Status is a custom type to represent the possible status
+type Status int
+
+const (
+	// Idle means the the service is avaiable to new processments
+	Idle Status = 0
```

Comment:
Não tínhamos combinado em PT_BR? Ou não decidimos?

Reply:
Não decidimos de vez, você não acha por 99% do código estar em inglês vale mais a pena isso ficar em inglês também?

Reply:
O repositório enriquecedores tem 30 linhas de código Golang. Não entendi bem de qual código está se referindo.

A struct Candidatura tem mais que isso e é em PT_BR. De toda forma, seja qual for a língua, precisamos decidir antes de colocar mais código no repo. Vou disparar a pergunta no grupo.

Reply:
A gente pode encerrar esse PR?